compiler/rustc_error_codes/src/error_codes/E0744.md

```diff
@@ -12,12 +12,7 @@ const _: i32 = {
 };
 ```
 
-At the moment, `if` and `match`, as well as the looping constructs `for`,
-`while`, and `loop`, are forbidden inside a `const`, `static`, or `const fn`.
+At the moment, `for` is forbidden inside a `const`, `static`, or `const fn`.
 
-This will be allowed at some point in the future, but the implementation is not
-yet complete. See the tracking issue for [conditionals] or [loops] in a const
-context for the current status.
-
-[conditionals]: https://github.com/rust-lang/rust/issues/49146
-[loops]: https://github.com/rust-lang/rust/issues/52000
+This may be allowed at some point in the future, but there is currently no
+plan to allow `for` in const contexts.
```

Comment:
That's not true, those are planned but blocked on https://github.com/rust-lang/rust/issues/67792. See https://rust-lang.github.io/const-eval/ - maybe we should link that in the `--explain` text?

Reply:
Alright, I should've looked into that.

Reply:
I'm not sure if I'm reading that diagram wrong but wouldn't it also be blocked on #57349?

Reply:
Sure, sounds right.